tools/lib/capitalization.py

```diff
@@ -157,6 +157,7 @@
     r"your-organization-url",
     # Used in invite modal
     r"or",
+    r"{user_name} created a new stream #\*\*{stream.name}\*\*",
```

Comment:
This may be better fixed by changing the regexs in this tool?

Reply:
Turns out it was just the `{stream.name}` part that was getting parsed as two sentences, I just added that bit to make it more clear what's going on. Possibly the sentence split regex should be changed, but I'm a bit wary of false negatives there, so I'm not making that change for now.